Development-Guide/Git-Guides-and-Standards/Commit-Messages.adoc

```diff
@@ -21,26 +21,28 @@ Further paragraphs come after blank lines.
   single space. Use a hanging indent.
 ```
 
-Your commit title should be phrased so it can answer the question "If this commit is applied, it will..."
+#### Subject line
 
-### Example commit message
+The subject line should prefixed with the issue number (if applicable), and be able to answer the question:
 
-```
-Change the units of cell voltages from mV to V
+_If applied, this commit will <your subject line here>_
 
-Based on testing and feedback....
-```
+#### Information in commit messages
 
-#### Subject line
+The summary should be in paragraph form and should discuss the _whys_ of the change, for example, why the specific bug occurred or why this particular change works.
 
-The subject line should be able to describe the effects of the patch:
+Documentation or references can be linked depending on how complicated the change is.
 
-_If applied, this commit will <your subject line here>_
+If the developer & reviewer agree that the change is trivial or simple enough that just subject the line is explanatory, then an extra paragraph is not needed.
```

Comment:
Lol I thought that was supposed to be 'that just the subject line is explanatory', or am I misunderstanding what this line is saying?

Reply:
I'm not an English major